dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -1152,6 +1152,18 @@
           # decision is evaluated. If the cache is full, the least used DRG gets evicted.
           # drgCacheCapacity: 1000
 
+        # jobs:
+          # Allows to configure the Job Timeout Trigger's batch limit. This is the number of jobs
```

Comment:
This is probably missing some more mature documentation, I tried my best according to my knowledge, but comparing to the one above it (from `messages`), it could use some help!